### PR TITLE
data: harshasingamshetty1

### PR DESCRIPTION
**Pull request by harshasingamshetty1**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3508"></a>

### CODE REVIEW

File: data/harshasingamshetty1.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "url": "https://www.linkedin.com/in/harsha-vardhan-singamshetty-3a1a911b1/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/harshasingamshetty1",
+      "icon": "FaGithub"
+    }
+    
+  ]
```
Comment (EmmaDawsonDev) [inline data/harshasingamshetty1.json:25]:
```suggestion
  ],
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
There's a missing comma on line 25, otherwise this looks good.

**Review by Tejender1521 — CHANGES_REQUESTED** [review]
Validate JSON file and put "," after links array

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**